index.html

```diff
@@ -0,0 +1,204 @@
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Join</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">News</a>
+          </li>
+        </ul>
+        <button class="btn btn-outline-success" type="submit">
+          SD Campaign
+        </button>
+      </div>
+    </div>
+  </nav>
+
+  <section class="first-section">
+    <div class="first-section-container">
+      <h3 class="second-heading">"Hello! Sharing world"</h3>
+      <h1 class="open-heading">SOFTWARE DEVELOPER GLOBAL SUMMIT 2021</h1>
+      <div class="first-section-paragraph">
+        <p class="first-paragraph">
+          O joyful celebration believing in the value of openness and
+          sharing,creating a positive change with people from over 80
+          xountries is taking place in October,in Korea.
+        </p>
+      </div>
+      <div class="event-details">
+        <h3>2021.10.20(MON) ~ 22(WED)</h3>
+        <p class="venue">
+          @ National Museum of Korea,Art Center Nabi and more
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="second-section">
+    <div class="second-section-container">
+      <h2 class="second-section-heading">Main <span>Pro</span>gram</h2>
+
+      <div class="program-list">
+        <div class="first-block flex-box">
+          <img class="icons" src="images/program1.png" alt="" />
+          <h2>Lecture</h2>
+          <p>
+            Listen to the spekaer from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+
+        <div class="second-block flex-box">
+          <img class="icons" src="images/program2.png" alt="" />
+          <h2>CC Exhibiton</h2>
+          <p>
+            Appreciate various creations applying CC licence of
+            artists,organized from Art Center Nabi.
+          </p>
+        </div>
+
+        <div class="third-block flex-box">
+          <img class="icons" src="images/program3.png" alt="" />
+          <h2>Forum</h2>
+          <p>
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+
+        <div class="fourth-block flex-box">
+          <img class="icons" src="images/program4.png" alt="" />
+          <h2>Workshop</h2>
+          <p>
+            Try creating your own work using open source licence rather than
+            just watching.
+          </p>
+        </div>
+
+        <div class="fifth-block flex-box">
+          <img class="icons" src="images/program5.png" alt="" />
+          <h2>CC Ignite</h2>
+          <p>
+            Get opportunities to network with CC affiliates around the
+            world,also after the summit.
+          </p>
+        </div>
```

Comment:
All these blocks have the same styling. perhaps for more readable code, they could just have one class program-block
```suggestion
        <div class="program-block flex-box">
          <img class="icons" src="images/program1.png" alt="" />
          <h2>Lecture</h2>
          <p>
            Listen to the speaker from various countries about the messages of
            sharing and opening.
          </p>
        </div>

        <div class="program-block flex-box">
          <img class="icons" src="images/program2.png" alt="" />
          <h2>CC Exhibiton</h2>
          <p>
            Appreciate various creations applying CC licence of
            artists,organized from Art Center Nabi.
          </p>
        </div>

        <div class="program-block flex-box">
          <img class="icons" src="images/program3.png" alt="" />
          <h2>Forum</h2>
          <p>
            Have the time to share your thoughts and opinions with experts for
            each topic.
          </p>
        </div>

        <div class="program-block flex-box">
          <img class="icons" src="images/program4.png" alt="" />
          <h2>Workshop</h2>
          <p>
            Try creating your own work using open source licence rather than
            just watching.
          </p>
        </div>

        <div class="program-block flex-box">
          <img class="icons" src="images/program5.png" alt="" />
          <h2>CC Ignite</h2>
          <p>
            Get opportunities to network with CC affiliates around the
            world,also after the summit.
          </p>
        </div>
```